process.c

```diff
@@ -8563,10 +8584,12 @@ proc_warmup(VALUE _)
     RB_VM_LOCK_ENTER();
```

Comment:
I wonder if we still actually need to lock the VM here, wasn't that for the transient heap?